.github/workflows/readme-linter.yml

```diff
@@ -0,0 +1,23 @@
+name: Lint plugin readmes
+on:
+#  push:
+#    branches-ignore: master
+  pull_request:
+    branches: # Names of target branches, not source branches
+      - master
+jobs:
+  run-readme-linter:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/setup-go@v3
+      - uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+      - name: Get changed files
+        id: changed-files
+        uses: tj-actions/changed-files@v18.7
+        with:
+          base_sha: ${{ github.event.pull_request.base.sha }}
+          files: plugins/**/README.md
```

Comment:
Should we include the stuff under docs and the base README?

Reply:
This linter's main purpose is to make sure plugin readmes follow a consistent format for the plugin type. Since the non-plugin markdown files (the base readme and everything in docs) don't follow a consistent format, this linter wouldn't be able to do much for them besides check for long lines.

The non-plugin markdown files are covered as generic markdown by super-linter/markdownlint in a different CI check. (the github action that shows up as "Lint Code Base") We should probably make that more strict and have it check line length.

Reply:
ah right I'm getting "markdown" linter and "readme" linter mixed up.

Why did you end up going the GH actions route versus circle ci?

Reply:
I don't have a strong preference. I partly chose GH actions to learn more about them. They seem nicer in a few ways- simpler to reuse existing actions to check out, set up go, determine which files changed.